src/components/NavBar/index.tsx

```diff
@@ -27,7 +27,7 @@ const NavBar: React.FC = () => {
       <FlexContainer>
         <Logo glyph="ParsleyLogo" size={24} useStroke />
         <LinkContainer>
-          <StyledLink href={wikiURL}>Wiki</StyledLink>
+          <StyledLink href={docsURL}>Documentation</StyledLink>
```

Comment:
![image](https://github.com/evergreen-ci/parsley/assets/4605522/b75aaada-15e7-4260-837d-dc6f22b684e2)
The text is a lot wider and messes up the navbar if a search is also applied.